docs/content/.vitepress/config.ts

```diff
@@ -131,61 +146,46 @@ function getSocialLinks() {
   return [Social.item(Github, GH_URL), Social.item(Npm, NPM_URL)]
 }
 
+type Frontmatter = { title: string }
```

Comment:
I would honestly move all that added stuff into `helpers.ts`, there's no reason to bloat the config.